source/_components/tradfri.markdown

```diff
@@ -48,3 +48,6 @@ Do not use the `api_key` variable. The key is only needed once at initial setup.
 Please make sure you have `autoconf` installed (`apt-get install autoconf`) if you want to use this component. Also, installing some dependencies might take considerable time (>1h) on slow devices. You might have to use `sudo` when installing `autoconf`.
 
 </p>
+
+## Troubleshooting
```

Comment:
Please change this into a linkable title:
```
### {% linkable_title Troubleshooting %}
```

Reply:
See [the standards guide](https://developers.home-assistant.io/docs/en/documentation_standards.html) for details on linkable titles, etc